spec/jobs/reports/combined_invoice_supplement_report_v2_spec.rb

```diff
@@ -578,6 +578,74 @@
     end
   end
 
+  describe '#combine_by_iaa_month' do
+    let(:service_provider) { create(:service_provider) }
+
+    let(:by_iaa_results) do
+      [
+        {
+          key: 'IAA123',
+          year_month: '202408',
+          iaa_start_date: '2024-08-01',
+          iaa_end_date: '2025-07-31',
+        },
+      ]
+    end
+
+    let(:by_issuer_results) do
+      [
+        {
+          iaa: 'IAA123',
+          issuer: service_provider.issuer,
+          year_month: '202408',
+        },
+        {
+          iaa: 'IAA123',
+          issuer: service_provider.issuer,
+          year_month: '202409',
+        },
+      ]
+    end
```

Comment:
Is it really just a mismatch on year_month between iaas and issuers? I thought the error was happening because an issuer year_month was outside of the range of the iaa.

Reply:
I feel like the result is the same no?

Reply:
It should be, it just makes more sense to me now.